Functions.Rmd

```diff
@@ -1028,13 +1027,13 @@ a <- 2
 (a <- 2)
 ```
 
-And this is what makes it possible to chain assignment:
+This is what makes chain assignment possible:
```

Comment:
I can't explain precisely why but this changes the meaning - maybe because it makes "chain assignment" feel like a compound noun, instead of being a verb-noun pair?